src/ResourceManager/Websites/Commands.Websites/Properties/Resources.resx

```diff
@@ -117,6 +117,9 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
+  <data name="RemovingWebHostPlan" xml:space="preserve">
+    <value>Are you sure you want to remove webhosting plan '{0}'</value>
```

Comment:
webhosting --> web hosting
also I think we need to inform that all sites will be removed as a result of this (I think)